bundle/filefs.go

```diff
@@ -0,0 +1,73 @@
+// +build go1.16
```

Comment:
💡 you're correct here, we don't need any non-1.16 stubs: the exported function will just _not be there_.